server/services/tables.js

```diff
@@ -24,20 +24,47 @@ const isTableNameTaken = async (name) => {
 
 const validateTable = async (tableName, password, username) => {
     try {
-        const table = await tableSchema.findOne({ "tableName": tableName, "password": password });
-        // add user to table
-        if(table) {
-          const user = await userSchema.findOne({ "username": username });
-            table.playersOnTable.push(user);
-            await table.save();
-        }
-        return table; // If user is found, authentication is successful
+
+      return await tableSchema.findOne({ "tableName": tableName, "password": password });
+  
```

Comment:
when validating we only entring the table into the modal but not joining and sitting